### PR TITLE
Add space, remove extra b.

### PR DESCRIPTION
**Pull request by marcgel**
Added small changes to README on main page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,15 @@ This is a user-space file system for interacting with [Google Cloud
 Storage][gcs]. It is not yet ready for public consumption, so please do not use
 it for anything of importance in its current state.
 
-[gcs]: https://cloud.google.com/storage/
+[gcs]: https://cloud.google.com/storage/docs
```
Comment (jacobsa) [inline README.md]: Is this really a better URL? The other one is a nice product overview, etc. It's clearly the landing page for GCS.

File: README.md [hunk 2]
```diff
@@ -2,15 +2,15 @@ This is a user-space file system for interacting with [Google Cloud
...
 
 
 # Installing
 
-See [installing.md][] for full installation instructions. The summary is that if
+See [installing.md][installing.md] for full installation instructions. The summary is that if
```
Comment (jacobsa) [inline README.md]: Why this change? The empty brackets thing is part of official markdown syntax, and renders fine on GitHub.

File: README.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ instructions [here][create-key] to create a service account, generate a private
 key, and download a JSON file containing the private key. Place the JSON file on
 the machine that will be mounting the bucket.
 
-[console]: console.developers.google.com
+[console]: https://console.developers.google.com/project/_/storage/browser
```
Comment (jacobsa) [inline README.md]: Thanks for fixing the broken link. This isn't the right bit of the console to point at though; the link is about generation credentials.

File: README.md (hunk 1 above)
Comment (marcgel) [inline README.md]: Okay to leave as is. I was thinking for the audience, landing on the marketing page is just one more click to get where they want.

File: README.md (hunk 2 above)
Comment (marcgel) [inline README.md]: My bad. I clicked it and thought it landed on the same page.  But I see it works fine.

File: README.md (hunk 3 above)
Comment (marcgel) [inline README.md]: Totally striking out here. I meant this as a help
https://console.developers.google.com/project/_/apiui/credential - which could get the user to pick the project and then dumpt them on the credential page if I understood the intent.

**Comment by marcgel** [comment]
Let's ditch this pull request for now and we'll get updates later. Ok?

**Comment by jacobsa** [comment]
Sounds good. By the way, if it winds up mattering in the future: my preference is granular pull requests. So don't feel bad about sending me one just to fix a typo or anything; I like that better than bundled unrelated changes.

**Comment by marcgel** [comment]
Ok, thanks.
